front/components/assistant/markdown/RenderMessageMarkdown.tsx

```diff
@@ -0,0 +1,457 @@
+import type { ReactNode } from "react";
```

Comment:
Anything that changed here? Can you flag?

Reply:
- CodeBlockWithExtendedSupport moved in dedicated file, along with SyntaxHighlighter and mermaid handling.
- SyntaxHighlighter loaded with React.lazy instead of next/dynamic . Since we're not using the ssr part, it's equivalent
- Mermaid support not using a context anymore, directly handling in CodeBlockWithExtendedSupport. Some layout changes in header (moving buttons on the left as they were overlapping the copy code button, header color for consistency/simplicity)
- Visualization support moved in dedicated file and not included by default . It's included through additionalMarkdownComponents and additionalMarkdownPlugins in AgentMessage . Content sanitization (support for older viz format) also moved AgentMessage
- Mention and Cite also moved in dedicated file - no change.